.drone.yml

```diff
@@ -0,0 +1,7 @@
+pipeline:
+  test:
+    image: lampepfl/dotty:latest
+    pull: true
+    commands:
+      - cd dotty && sbt publishLocal
+      - sbt community/test
```

Comment:
It was `sbt test` at first but that seemed run all the tests in `dotty`, too. I have now moved the community build code into a `dotty-community-build` subdirectory so that all top-level directory names match a repo name.